src/sql/workbench/contrib/notebook/browser/models/notebookContexts.ts

```diff
@@ -105,43 +106,10 @@ export class NotebookContexts {
 			activeConnections = [];
 			connections.forEach(connection => activeConnections.push(connection));
 		}
-		if (defaultConnection === NotebookContexts.DefaultContext.defaultConnection) {
```

Comment:
Seemed like this section is only reachable if the Active Connections include DefaultContext.defaultConnection, but before these changes that field was private. So I've gone ahead and removed this section.